### PR TITLE
Omnipod Dash Documentation

### PR DESCRIPTION
**Pull request by AmineT1D**
Initial Dash documentation, based on EROS documentation with tweaks.
Helpfull links :
[Photos](https://drive.google.com/drive/folders/1j8J45_5q2Bx6HEDaBwyS6-rp97kQGMpe?usp=sharing)
[Documentation with photos .rst format](https://docs.google.com/document/d/1iONZ_uvilgS7yrVDgwNSTiFN1qOLfBiikjL4KbHgvJ0/edit#)

### CODE REVIEW

File: docs/EN/Configuration/OmnipodDASH.md [hunk 1]
```diff
@@ -1,91 +1,567 @@
 
-# About this documentation
+About this documentation
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:2]: We are keeping this while working on the docs or do we plan to keep it forever?
If it's temporary, maybe we could move it in a comment in the PR?

File: docs/EN/Configuration/OmnipodDASH.md [hunk 2]
```diff
@@ -1,91 +1,567 @@
...
+========================
 
-Initial conten bwelow was based on [a rough draft on Google docs](https://docs.google.com/document/d/1iONZ_uvilgS7yrVDgwNSTiFN1qOLfBiikjL4KbHgvJ0/edit)
+Initial conten Below is based on Google doc drafts :
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:5]:
```suggestion
Initial content Below is based on Google doc drafts :
```

File: docs/EN/Configuration/OmnipodDASH.md [hunk 3]
```diff
@@ -1,91 +1,567 @@
...
 
-### Enabling the Omnipod DASH Driver in AAPS
+*This does NOT mean you should throw away your PDM, it is recommended to keep it around as a backup and for emergencies, for instance when your phone gets lost or AAPS is not working correctly.*
 
-### Omnipod Configuration
+**Your pod will not stop delivering insulin when it is not connected to AndroidAPS**.
+Default basal rates are programmed on the pod on activation as defined in the current active profile.
+As long as AndroidAPS is operational it will send basal rate commands that run for a maximum of 30 minutes. When for some reason the pod does not receive any new commands (for instance because communication was lost) the pod will automatically fall back to default basal rates.
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:64]: I think I saw AAPS sending TBRs or 60min or more.

File: docs/EN/Configuration/OmnipodDASH.md [hunk 4]
```diff
@@ -1,91 +1,567 @@
...
+|Resume_3|
+
+3. The **DASH** tab will update the **Pod status (1)** field to display **RUNNING,** and the **Resume Delivery** button will now display the **SUSPEND (2)** button.
+
+
+|Resume_4|
+
+## Acknowledging Pod Alerts
+
+*NOTE - The Silence ALERTS button is only available on the **DASH** tab when the pod expiration or low reservoir alert has been triggered.*
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:246]: If the `Silence ALERTS` button is not visible and you hear beep sounds from the pod, try to 'Refresh pod status'.

File: docs/EN/Configuration/OmnipodDASH.md [hunk 5]
```diff
@@ -1,91 +1,567 @@
...
+4. Go to the **Omnipod (POD)** tab. Under the **Active Pod alerts** field, the warning message is no longer displayed, and the active pod will no longer issue pod expiration warning beeps.
+
+## View Pod History
+
+This section shows you how to review your active pod history and filter by different action categories. The pod history tool allows you to view the actions and results committed to your currently active pod during its three days (72 - 80 hours) life.
+
+This feature is helpful in verifying boluses, TBRs and basal commands that were sent to the pod. The remaining categories are useful for troubleshooting issues and determining the order of events that occurred leading up to a failure.
+
+*NOTE:*
+**Uncertain** commands will appear in the pod history. However, due to their nature, you cannot ensure their accuracy.
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:272]: On Dash only the last command can be uncertain.

We are not sending new commands until the last 'uncertain' command becomes 'confirmed' or 'denied'. The way to 'fix' uncertaing commands is to 'refresh pod status'.

File: docs/EN/Configuration/OmnipodDASH.md [hunk 6]
```diff
@@ -1,91 +1,567 @@
...
+*NOTE: If any message in the **DASH** tab status fields report (uncertain), then you will need to press the Refresh button to clear it and refresh the pod status.*
+
+|DASH_Tab_1|
+
+## Fields
+
+* **Bluetooth Address:** Displays the current bluetooth address of the connected Pod.
+* **Bluetooth Status:** Displays the current connection status.
+* **Connection quality:** Displays the BT connection quality.
+* **Delivery Status:** Displays the insulin delivery status.
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:295]: This is visible only when`engineering_mode` is enabled.

File: docs/EN/Configuration/OmnipodDASH.md [hunk 7]
```diff
@@ -1,91 +1,567 @@
...
+# Troubleshooting
+
+## Pod Failures
+
+Pods fail occasionally due to a variety of issues, including hardware issues with the Pod itself. It is best practice not to call these into Insulet, since AAPS is not an approved use case. A list of fault codes can be (found here](https://github.com/openaps/openomni/wiki/Fault-event-codes) to help determine the cause.
+
+## Preventing error 49 pod failures
+
+This failure is related to an incorrect pod state for a command or an error during an insulin delivery command. This is when the driver and Pod disagree on the actual state. The Pod (out of a build-in safety measure) then reacts with an unrecoverable error code 49 (0x31) ending up with what is know as a “screamer”: the long irritating beep that can only be stopped by punching a hole at the appropriate location at the back of the Pod.
+The exact origin of a “49 pod failure” often is hard to trace. In situations that are suspected for this failure to occur (for instance on application crashes, running a development version or re-installation), the best way to prevent the error is to stop any communication with the Pod using the “SUSPEND POD” command in advance.
```
Comment (avereha) [inline docs/EN/Configuration/OmnipodDASH.md:452]: We might remove the 'Suspend' button.

**Review by avereha — APPROVED** [review]
Great work!

I left a few small comments/details.
There are two points that I'm not sure if they are going to change or not yet:
  * we might remove the 'suspend' button and functionality
  * the Connection Quality field might be enabled for all users or only when `engineering_more` is enabled.

File: docs/EN/Configuration/OmnipodDASH.md (hunk 1 above)
Comment (AmineT1D) [reply docs/EN/Configuration/OmnipodDASH.md:2]: Yep we can move it to comments

**Pull request merged**